src/rgw/rgw_common.h

```diff
@@ -20,6 +20,11 @@
 #include <string_view>
 #include <atomic>
 #include <unordered_map>
+#include <fmt/format.h>
+
+#undef FMT_HEADER_ONLY
+#define FMT_HEADER_ONLY 1
+#include <fmt/format.h>
```

Comment:
was that first `#include` line intentional?

Reply:
@cbodley it's not. thanks for pointing this out. i corrected it in the latest revision. hopefully it is able to address link failure.